build_manager.go

```diff
@@ -89,30 +89,26 @@ func (b *Bob) maybeQueue(path string) {
 		return
 	}
 
+	// clear scroll buffer
+	print("\033c")
+
+	// setup list of commands
+	v := vow.To("go", "build", "./...")
```

Comment:
Since we're just chaining these commands, can we just chain the calls as `vow.To(...).Then(...).Then(...)`